src/views/LoginSuccess.vue

```diff
@@ -78,18 +78,18 @@ export default class LoginSuccess extends Vue {
 
         this.storeAndUpdateResult();
 
-        if (this.keyguardResult.keyType === KeyStorageType.BIP39) {
+        if (this.keyguardResult.keyType === WalletType.BIP39) {
             // Init Keyguard iframe to derive addresses
             this.keyguard = new KeyguardClient();
         }
 
-        if (this.keyguardResult.keyType === KeyStorageType.LEDGER) {
-            if (this.keyguardResult.keyType === KeyStorageType.LEDGER) this.walletLabel = 'Ledger Wallet';
+        if (this.keyguardResult.keyType === WalletType.LEDGER) {
+            if (this.keyguardResult.keyType === WalletType.LEDGER) this.walletLabel = 'Ledger Wallet';
```

Comment:
I'm not 100% clear on this one. Is it coming from the KG as the name suggests? Should the KG know about keys that are on the Ledger?

Reply:
You are correct, the Keyguard is not involved when dealing with Ledger wallets. This will be fixed when dealing with #18.